roles/automount/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+file_server: "://example.server.org/"
+project_base: /opt
+mount_point: "{{ project_base }}/ingest"
+mount_opts: -fstype=cifs,ro,sec=ntlm,noexec,_netdev
+automount_user:
```

Comment:
Does a null value work here? I added "test" as a value for both user and pw on my last test run - filling in all variables is a habit for me, but leaving a null suggests that the value is/should be a secret.